committees/project-director-election-process.md

```diff
@@ -0,0 +1,34 @@
+# Leadership Council _Project Director Election Process_ Committee
+
+## Goals
+
+* Design the election process for the next set of Rust Foundation project directors (ratified by the leadership council)
+* Clearly document and communicate the process and support its execution
+* Track, review, and document the effectiveness of the process
+
+## Delegated Responsibilities
+
+* None, this committee exists to create a proposal which will be consented to by the council
```

Comment:
@jackh726 This section is mostly intended for listing which decision making powers the committee has been delegated. In this case, they have none because all they are doing is preparing a proposal. 

Would something like the following be a little clearer?

> This committee has no decision-making authority. It's primary responsibility is to create a proposal for the Council on the Project Director selection process.

Reply:
I'm interpreting Jack's comment as pointing out that deciding what to put into the proposal is a form of power, which I agree with.

I'm changing the wording of that section to this:

> * This committee decides the content of the proposal which will be presented to the council for their consent